### PR TITLE
Add comments and minor changes to realtimerobotclient.py

### PR DESCRIPTION
**Pull request by felixvd**
No logic should change in this PR. These changes align `realtimerobotclient` more closely with the generated version.

### CODE REVIEW

**Comment by felixvd** [comment]
Should be merged via `Squash and Merge`

File: python/mujincontrollerclient/realtimerobotclient.py [hunk 1]
```diff
@@ -1372,35 +1450,39 @@ def GetState(self, timeout=10, usewebapi=None, fireandforget=False, **kwargs):
         """
 
         Args:
-            timeout (float, optional):  (Default: 10)
+            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 4.0)
```
Comment (yoshikikanemoto) [inline python/mujincontrollerclient/realtimerobotclient.py:1453]: default still looks 10sec

File: python/mujincontrollerclient/realtimerobotclient.py [hunk 2]
```diff
@@ -1213,56 +1277,60 @@ def RunDynamicsIdentificationTest(self, timeout, usewebapi=False, **kwargs):
         """
 
         Args:
-            timeout:
-            usewebapi (bool, optional): If True, send command through Web API. Otherwise, through ZMQ. (Default: False)
+            timeout (float): Time in seconds after which the command is assumed to have failed. (Default: 4.0)
```
Comment (yoshikikanemoto) [inline python/mujincontrollerclient/realtimerobotclient.py:1280]: there is no default value

File: python/mujincontrollerclient/realtimerobotclient.py [hunk 3]
```diff
@@ -23,6 +23,14 @@ def __init__(self, robotname, robotspeed=None, robotaccelmult=None, envclearance
             robotaccelmult (str, optional): Current robot acceleration multiplication.
             envclearance (str): Environment clearance in millimeter, e.g. 20
             robotBridgeConnectionInfo (str, optional): dict holding the connection info for the robot bridge.
+            taskzmqport (int): Port of the task's ZMQ server, e.g. 7110
```
Comment (yoshikikanemoto) [inline python/mujincontrollerclient/realtimerobotclient.py:26]: can you also explicitly take and pass these documented arguments?

**Comment by rdiankov** [comment]
thanks

**Pull request merged**